lib/teiserver_web/templates/moderation/report/index.html.heex

```diff
@@ -85,7 +77,7 @@
 
             <td>
               <a
-                href={Routes.profile_overview_path(@conn, :overview, report.target_id)}
+                href={Routes.moderation_report_path(@conn, :user, report.target_id)}
```

Comment:
Similar, I think using the `~p` sigil is preferred, because you get compilation warnings (errors?) if they don't match anything:
```
              <a href={~p"/moderation/report/user/#{report.reporter_id}"}>
```